tests/bases.py

```diff
@@ -1,7 +1,23 @@
+from flask.testing import FlaskClient
+
 from app import create_app, db
 from app.models import Framework, FrameworkLot
 
 
+class TestClient(FlaskClient):
+
```

Comment:
Please can we inline document what this is doing, why it is doing and what a impact it has for a developer when writing tests

Reply:
Will do

Reply:
Done 👍